tap-snapshots/test-lib-fund.js-TAP.test.js

```diff
@@ -5,7 +5,11 @@
  * Make sure to inspect the output below.  Do not ignore changes!
  */
 'use strict'
-exports[`test/tap/fund.js TAP fund containing multi-level nested deps with no funding > should omit dependencies with no funding declared 1`] = `
+exports[`test/lib/fund.js TAP fund a package with type and multiple sources > should print name only 1`] = `
+
```

Comment:
Curious, this says "should print name only", but it's not printing anything?

Reply:
oooooops, will review what's going on 😬